api/src/main/java/org/openmrs/ui/framework/UiUtils.java

```diff
@@ -397,7 +397,20 @@ public String formatDatePretty(Date date) {
 			return format(dateExt.getDateWithoutTime());
 		}
 	}
-	
+
+	/**
+	 * Formats a date without any timezone conversion
+	 *
+	 * @param date the date to format
+	 * @return date formatted
+	 */
+	public String formatDateWithoutTimezoneConversion(Date date) {
+		String dateFormatGP = Context.getAdministrationService()
+				.getGlobalProperty(UiFrameworkConstants.GP_FORMATTER_DATE_FORMAT, "dd.MMM.yyyy");
+		SimpleDateFormat dateFormat = new SimpleDateFormat(dateFormatGP, Context.getLocale());
```

Comment:
and what if we just use 	SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");

Reply:
That is kinda what we are doing but we are using the GP, so everyone could have their own format, and we are passing the locale, so, if the format of the months is "MMM" we could translate to their language.

Reply:
Because, we have a GP that specifies that format, so I think we should use it. If I am missing something, please let me know.

Reply:
> That is kinda what we are doing but we are using the GP, so everyone could have their own format,

this makes sense to me